tests/testthat/test-db_FRED.R

```diff
@@ -27,6 +27,23 @@ testthat::test_that("upercase group quarters table", {
   testthat::expect_equal(class(gq)[[1]], "data.frame")
 })
 
+# testthat::test_that("test group quarters table", {
+#   gq <- import_from_FRED(
+#     table_name = "GQ_UNIT",
+#     prod = FALSE,
+#     uid = httr2::secret_decrypt("QUHBRb_yoy2RRj59qno8NVXA7mW402xkins", "COUNCILR_KEY"),
+#     pwd = httr2::secret_decrypt("tsGlVlM0KBdLcUPNLX5f0Hll2_6HFF_rQx62cw", "COUNCILR_KEY"),
```

Comment:
I've not used `httr2::secret_decrypt`. Is there a reason that we wouldn't use keychain here too?

Reply:
testthat environments are very picky - I've found that this works across machines and lets us use an environment variable in Github Actions for testing.